docs/dom-testing-library/api-events.md

```diff
@@ -93,3 +93,31 @@ const myEvent = createEvent.click(node, { button: 2 })
 fireEvent(node, myEvent)
 // myEvent.timeStamp can be accessed just like any other properties from myEvent
 ```
+
+## Using Jest Function Mocks
+[Jest's Mock functions](https://jestjs.io/docs/en/mock-functions) can be used to test that a callback passed to the function was called, or what it was called when the event that **should** trigger the callback function does trigger the bound callback.
+
+```jsx
+const Button = ({onClick,children}) => <button onClick={onClick}>{children}</button>
+describe('Button component', () => {
+    let onClick;
```

Comment:
Sharing this variable through a bunch of tests makes it harder to maintain long-term. This isn't quite the same thing, but it's similar to what I talk about in [Test Isolation with React](https://kentcdodds.com/blog/test-isolation-with-react).